openwakeword/run.sh

```diff
@@ -1,3 +1,2 @@
 #!/usr/bin/env bash
-python3 -m wyoming_openwakeword \
-    --uri 'tcp://0.0.0.0:10400' "$@"
+python3 -m wyoming_openwakeword --uri 'tcp://0.0.0.0:10400' "$@"
```

Comment:
Why did you change it to single line? Seems unrelated and reduces legibility

Reply:
oh I did this because if I don't it won't run the script. its not recognizing the / I think this is a different in my environment refactoring it ensures its environment agnostic.

Reply:
Weird, this is supposed to runs inside the container, should not care about your environment.